recipe/meta.yaml

```diff
@@ -24,13 +28,21 @@ requirements:
 
 test:
     commands:
-        # Verify libraries.
-        - test -f $PREFIX/lib/libflann_s.a                                        # [unix]
-        - test -f $PREFIX/lib/libflann.dylib                                      # [osx]
-        - test -f $PREFIX/lib/libflann.so                                         # [linux]
-        - test -f $PREFIX/lib/libflann_cpp_s.a                                    # [unix]
-        - test -f $PREFIX/lib/libflann_cpp.dylib                                  # [osx]
-        - test -f $PREFIX/lib/libflann_cpp.so                                     # [linux]
+        - conda inspect linkages -p $PREFIX flann  # [not win]
+        - conda inspect objects -p $PREFIX flann   # [osx]
+        # Verify libraries exist
+        - test -d $PREFIX/include/flann                       # [not win]
+        - test -f $PREFIX/lib/libflann_s.a                    # [unix]
+        - test -f $PREFIX/lib/libflann.dylib                  # [osx]
+        - test -f $PREFIX/lib/libflann.so                     # [linux]
+        - test -f $PREFIX/lib/libflann_cpp_s.a                # [unix]
+        - test -f $PREFIX/lib/libflann_cpp.dylib              # [osx]
+        - test -f $PREFIX/lib/libflann_cpp.so                 # [linux]
+        - if not exist %LIBRARY_LIB%\\flann_cpp_s.lib exit 1  # [win]
+        - if not exist %LIBRARY_LIB%\\flann_s.lib exit 1      # [win]
+        - if not exist %LIBRARY_LIB%\\flann.lib exit 1        # [win]
+        - if not exist %LIBRARY_BIN%\\flann.dll exit 1        # [win]
+        - if not exist %LIBRARY_INC%\\flann exit 1            # [win]
```

Comment:
We could tidy these a bit to keep the easier to maintain.

ref: https://github.com/conda-forge/boost-cpp-feedstock/blob/bae4ccf3b9ea2be3b452de3628902b22919241f5/recipe/meta.yaml#L50-L91